File: VeamIt-project/Android/veam31000287/veam31000287/src/main/java/co/veam/veam31000287/ConsoleTutorialElement.java
package co.veam.veam31000287;

/**
 * Created by veam on 12/7/16.
 */
public class ConsoleTutorialElement {
    public String tutorialElementId ;
    public String title ;
    public String description ;
    public String imageFileName ;
    public int kind ;

    public ConsoleTutorialElement(String tutorialElementId, String title, String description, String imageFileName, int kind) {
        this.tutorialElementId = tutorialElementId;
        this.title = title;
        this.description = description;
        this.imageFileName = imageFileName;
        this.kind = kind;
    }
}

File: VeamIt-project/iOS/veam00000000/ConsoleTutorialElement.m
//
//  ConsoleTutorialElement.m
//  veam00000000
//
//  Created by veam on 2/17/15.
//  Copyright (c) 2015 veam. All rights reserved.
//

#import "ConsoleTutorialElement.h"

@implementation ConsoleTutorialElement

@synthesize tutorialElementId ;
@synthesize title ;
@synthesize description ;
@synthesize imageFileName ;
@synthesize kind ;

- (id)initWithTutorialElementId:(NSString *)targetTutorialElementId title:(NSString *)targetTitle description:(NSString *)targetDescription imageFileName:(NSString *)targetImageFileName kind:(NSInteger)targetKind
{
    self = [super init] ;
    if (self != nil){
        self.tutorialElementId = targetTutorialElementId ;
        self.title = targetTitle ;
        self.description = targetDescription ;
        self.imageFileName = targetImageFileName ;
        self.kind = targetKind ;
    }
    return self ;
}

@end
